app/frontend/src/assets/scss/variables.scss

```diff
@@ -0,0 +1,7 @@
+// Variables
+$bcgov-blue: #003366;
+$bcgov-blue-secondary: #38598a;
+$bcgov-orange: #fcba19;
+$bcgov-grey: #efefef;
+$bcgov-font: #494949;
+$bcgov-error: #D8292F;
```

Comment:
moving  these into a separate variables file, so we can use them from individual components